exp-models/addon/models/experiment.js

```diff
@@ -7,10 +7,13 @@ import JamModel from '../mixins/jam-model';
 
 export default DS.Model.extend(JamModel, {
     title: DS.attr('string'),
-    description: DS.attr('string'),
-    active: DS.attr('boolean'),
-    structure: DS.attr(),
-    permissions: DS.attr(),
+	description: DS.attr('string'),
```

Comment:
Very very tiny, but could you indent 4 spaces for consistency with our code style conventions elsewhere?